### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by Ostter**
update dependencies with vite and vitest

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,53 @@
-# Fullstack Boilerplate
+# Fullstack Boilerplate Sanic/React
 
-Use it for setting up a take-home interview assignment or a portfolio project.
+A backbone for your coding challenge.
```
Comment (sumeetbajra) [inline README.md:3]: Sounds very poetic 😄 
Just curious, did we use it from another boilerplate template?

**Review by sumeetbajra — APPROVED** [review]
The app works good, but just had question about the Readme file update.

**Review by ehsan-alva — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (Ostter) [reply README.md:3]: ye, borrowed it from another challenge

**Pull request merged**